packages/sources/coinmetrics/README.md

```diff
@@ -57,50 +57,41 @@ Endpoint to get the reference price of the asset.
 }
 ```
 
-## Burned Endpoint
+## Burned ETH Endpoint
```

Comment:
We should not be replacing this endpoint. Rather, we should create a new endpoint that utilizes the existing one.